sysutils/checkssl/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/szazeski/checkssl 0.2 v
+revision            0
+
+homepage            https://www.checkssl.org
+
+description         checkssl - simple command line tool to check or monitor your https certificate
+
+long_description    ${description}
+
+categories          sysutils devel
+platforms           darwin
+license             MIT
+installs_libs       no
+
+maintainers         {breun.nl:nils @breun} \
+                    openmaintainer
+
+checksums           rmd160  da8f7afd1cf684d329546ca2e69f38dda6791375 \
+                    sha256  270ae3b3ed60827aeafc89da54e00c066c5bd7daa2962bdc82e191ffd24c2fd1 \
+                    size    4071
+
+build.args          checkssl.go
```

Comment:
Though it works currently, this should not be required. Go understands the main package implicitly if it is present in the current directory.

Reply:
I remembered the build failing without this, but I just removed it and it indeed still worked, so I'll remove this line.